microsoft-edge/webview2/release-notes.md

```diff
@@ -31,6 +31,146 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 
+<!-- ====================================================================== -->
+## 1.0.1054.31
+
+Release Date: November 29, 2021
+
+[NuGet package for WebView2 SDK 1.0.1054.31](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1054.31)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 96.1054.31 or higher.
+
+### General
+
+*   General reliability fix.
+
+#### Bug fixes
+
+*   Turned off the Control-flow Enforcement Technology (CET) Shadow Stack feature for v96 WebView2 Runtime.
+*   Fixed an issue that was causing slow startup times when launching in a .NET single-file application.
+*   Fixed a crash caused by Microsoft Edge browser policies getting incorrectly applied to WebView2 as well.
+*   Fixed a crash that occurred when a pop-up window with a download dialog was closed.
+
+#### Promotions
+
+*  ICoreWebView2_7 (ADDED:)
+   *  PrintToPdf
+   *  CreatePrintSettings
+*  ICoreWebView2PrintToPdfCompletedHandler (ADDED)
+*  ICoreWebView2PrintSettings (ADDED:)
+   *  get_Orientation
+   *  put_Orientation
+   *  get_ScaleFactor
+   *  put_ScaleFactor
+   *  get_PageWidth
+   *  put_PageWidth
+   *  get_PageHeight
+   *  put_PageHeight
+   *  get_MarginTop
+   *  put_MarginTop
+   *  get_MarginBottom
+   *  put_MarginBottom
+   *  get_MarginLeft
+   *  put_MarginLeft
+   *  get_MarginRight
+   *  put_MarginRight
+   *  get_ShouldPrintBackgrounds
+   *  put_ShouldPrintBackgrounds
+   *  get_ShouldPrintSelectionOnly
+   *  put_ShouldPrintSelectionOnly
+   *  get_ShouldPrintHeaderAndFooter
+   *  put_ShouldPrintHeaderAndFooter
+   *  get_HeaderTitle
+   *  put_HeaderTitle
+   *  get_FooterUri
+   *  put_FooterUri
+   *  get_UserDataFolder
+
+
+<!-- ====================================================================== -->
+## 1.0.1083-prerelease
```

Comment:
@jasonstephen15 - is this section in the standard location, or should it be moved up for consistency?

Reply:
moved